BUILDING.md

```diff
@@ -81,7 +81,7 @@ installed before you can build it.
 - setuptools
 - pygobject
 - gtk3 (>= 3.12)
-- python (>= 2.7.4)
+- python (>= 3.8)
```

Comment:
This should be fine since even [supported LTS Debian (Bullseye)](https://endoflife.date/debian) ships at [least 3.9](https://wiki.debian.org/Python).